pkg/parser/task/command/cache.go

```diff
@@ -39,20 +41,25 @@ func NewCacheCommand(mergedEnv map[string]string, boolevator *boolevator.Booleva
 	})
 
 	folderSchema := schema.String("Path of a folder to cache.")
-	cache.RequiredField(nameable.NewSimpleNameable("folder"), folderSchema, func(node *node.Node) error {
+	cache.OptionalField(nameable.NewSimpleNameable("folder"), folderSchema, func(node *node.Node) error {
```

Comment:
I think you can use the logic that was before the push with expansion of HOME and stuff. Just extract the func in a variable and reuse for both folder and folders filelds. And you can use a single folders slice variable and check if it's not empty for the validation.

Reply:
> I think you can use the logic that was before the push with expansion of HOME and stuff.

I'm already doing this here https://github.com/cirruslabs/cirrus-cli/pull/421/files#diff-d9b28f74aa650a725937c6d5795b22f5d217930cc229655e2841c6bf11373a4eR121-R135.

>And you can use a single folders slice variable and check if it's not empty for the validation.

Not really, because you won't be able to catch cases where both `folder` and `folders` are set.